duecredit/collector.py

```diff
@@ -195,20 +197,65 @@ def cite(self, entry, **kwargs):
 
         return citation
 
+    @staticmethod
+    def _args_match_conditions(conditions, *fargs, **fkwargs):
+        """Helper to identify when to trigger citation given parameters to the function call
+        """
+        hit_default = False
+        for (arg, kwarg), values in iteritems(conditions):
```

Comment:
On Mon, 20 Jul 2015, Matteo Visconti di Oleggio Castello wrote:
>  > +    @staticmethod
>  > +    def _args_match_conditions(conditions, *fargs, **fkwargs):
>  > +        """Helper to identify when to trigger citation given parameters to the function call
>  > +        """
>  > +        hit_default = False
>  > +        for (arg, kwarg), values in iteritems(conditions):

>    maybe rename arg into argpos?

I think it is ok as is since args, kwargs are common abbreviations for
those, and it is nice to have loops of 

for var in vars:
 ;-)